### PR TITLE
Add LicAPI

### LINKED ISSUE #841: Add LicAPI
[https://github.com/m1guelpiedrafita/LicAPI/](https://github.com/m1guelpiedrafita/LicAPI).
[Demo](https://demo.miguelpiedrafita.com/LicAPI).
`PHP`

### PR DESCRIPTION
**Pull request by m1guelpf**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

Fix #841

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -724,6 +724,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [Flox](https://github.com/devfake/flox) `⚠` - Self hosted movie watch list with a rating based on a 3-point system for good, medium and bad. ([Demo](http://80.240.132.120/flox-demo/public/)) `MIT` `PHP`
   * [GeneWeb](https://geneweb.tuxfamily.org/wiki/GeneWeb) - GeneWeb is an open source genealogy software written in OCaml. It comes with a Web interface and can be used off-line or as a Web service. ([Demo](https://demo.geneweb.tuxfamily.org/gw7/), [Source Code](https://github.com/geneweb/geneweb)) `GPLv2` `OCAML`
   * [King Phisher](https://github.com/securestate/king-phisher) - King Phisher is a tool for testing and promoting user awareness by simulating real world phishing attacks. `BSD` `Python`
+  * [LicAPI](https://projects.miguelpiedrafita.com/LicAPI) - A PHP API to manage license info ([Source Code](https://github.com/m1guelpf/LicAPI), [Demo](https://demo.miguelpiedrafita.com/LicAPI)) `MPLv2` `PHP`
```
Comment (AndyR207) [inline README.md]: Source Code and Demo links are the wrong way around. You're also missing a full stop after the description.

**Comment by m1guelpf** [comment]
@AndyR207 @nodiscc Done!

**Comment by m1guelpf** [comment]
@n8225 @nodiscc @AndyR207 Done!

**Comment by AndyR207** [comment]
The demo does work? I think the better solution would just be to add the demo link as https://demo.miguelpiedrafita.com/LicAPI/admin.php

**Comment by n8225** [comment]
@AndyR207 https://demo.miguelpiedrafita.com/LicAPI/ doesn't work for me, I get a blank page no error message.  I think it is best without a demo link as the demo login credentials are on the top level homepage.  https://demo.miguelpiedrafita.com/LicAPI/admin.php doesn't do much good without the credentials.

**Pull request merged**